index.js

```diff
@@ -168,6 +170,7 @@ class K8sVMExecutor extends Executor {
         this.baseImage = this.kubernetes.baseImage;
         this.buildTimeout = this.kubernetes.buildTimeout || DEFAULT_BUILD_TIMEOUT;
         this.maxBuildTimeout = this.kubernetes.maxBuildTimeout || MAX_BUILD_TIMEOUT;
+        this.terminationGracePeriodSeconds = this.kubernetes.terminationGracePeriodSeconds || 60;
```

Comment:
is 60 default k8s value for `terminationGracePeriodSeconds`

Reply:
Default is 30 second, so we have kept it 60 default to test the entire termination setup